### ISSUE TITLE
Can this work with yolo?

### ISSUE THREAD

**Issue by iAmJuan550** [xingyizhou/CenterTrack#197]
I am using CenterTrack to track objects from the COCO dataset but the detection accuracy of your pre-trained network is too low. Is it possible to use yolo instead of centernet for detection?

**Comment by iAmJuan550** [comment]
Nevermind. I reread the paper and realized this is not tracking by detection.
